File: app/src/main/java/com/app/lsquared/ui/WebActivity.java
package com.app.lsquared.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.widget.FrameLayout;
import android.widget.RelativeLayout;

import com.app.lsquared.databinding.ActivityWebBinding;

public class WebActivity extends AppCompatActivity {

    ActivityWebBinding binding;

    @SuppressLint("ResourceType")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityWebBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        RelativeLayout rl = new RelativeLayout(this);

        FrameLayout frameLayout = new FrameLayout(this);
        frameLayout.setId(1);
        rl.addView(frameLayout);

//        binding.webRelativeLayout.addView(rl);
//        getSupportFragmentManager().beginTransaction().add(frameLayout.getId(), new FragmentNews()).commit();

    }
}

File: app/src/main/java/com/app/lsquared/WebActivity.kt
package com.app.lsquared

import android.os.Bundle
import android.webkit.WebChromeClient
import androidx.appcompat.app.AppCompatActivity
import com.app.lsquared.databinding.ActivityWebBinding

class WebActivity : AppCompatActivity() {

    private var binding: ActivityWebBinding? = null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityWebBinding.inflate(layoutInflater)
        setContentView(binding!!.getRoot())

        binding!!.webView.webChromeClient = WebChromeClient()
        binding!!.webView.settings.javaScriptEnabled = true
        binding!!.webView.loadUrl("https://constant-media.biodigital.com/")


    }
}